src/OpenLoco/src/Interop/Hooks.cpp

```diff
@@ -406,7 +407,7 @@ static uint32_t STDCALL lib_WriteFile(
     uintptr_t)
 {
     *lpNumberOfBytesWritten = fwrite(buffer, 1, nNumberOfBytesToWrite, hFile);
-    Console::logVerboseDeprecated("WriteFile(%s)", buffer);
+    Logging::verbose("WriteFile({})", fmt::ptr(buffer));
```

Comment:
Note here: Since it hooks WriteFile this would print arbitrary data, I changed it to print a pointer instead.